### PR TITLE
feat(hooks): namespace feature added

### PR DESCRIPTION
**Pull request by stevenp2**
- scope/namespace added 
- note that logging into codeartifact and using this project will now require `aws codeartifact login --tool npm --domain <domain> --repository <repo> --namespace @<scope/namespace>`

### CODE REVIEW

File: apps/hooks/pipeline/code-artifact-docker/index.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { InvocationRequest } from 'aws-sdk/clients/lambda';
 interface CAHandlerEvent extends InvocationRequest {
   codeArtifactRepo: string;
   codeArtifactDomain: string;
+  codeArtifactNamespace: string;
```
Comment (razin99) [inline apps/hooks/pipeline/code-artifact-docker/index.ts:9]: probably don't need this, since we read the namespace from `process.env`

**Review by razin99 — APPROVED** [review]
LGTM

File: apps/hooks/pipeline/code-artifact-docker/index.ts (hunk 1 above)
Comment (stevenp2) [reply apps/hooks/pipeline/code-artifact-docker/index.ts:9]: true

Comment (razin99) [reply apps/hooks/pipeline/code-artifact-docker/index.ts:9]: srry im dumb, we need this

**Pull request merged**